frontend/src/redux/slices/users/userManagementSlice.ts

```diff
@@ -11,8 +11,8 @@ export type UserData = {
 } & UpdateUser;
 
 export type UserQuearyArgs = {
-  includeTanks: boolean;
-  isEmployee: boolean;
+  includeTanks: boolean | undefined;
+  isEmployee: boolean | undefined;
```

Comment:
this should be another type

Reply:
sorry includeTanks should just be boolean

Reply:
done!